lib/tasks/pot.rake

```diff
@@ -27,19 +27,31 @@ task :pot do
 end
 
 namespace :check do
+
+  def interpolation_message
+    "Note: \#{foo} substitution in translatable strings does\n" \
+    "not work properly, use\n" \
+    "  _(\"foo %{bar} baz\") % { :bar => bar }\n" \
+    "or\n" \
+    "  _(\"foo %s baz\") % bar\n\n"
```

Comment:
when I see all that escaping and line endings and so on, can we here use simple heredoc `<<~`